jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -121,10 +122,8 @@ def load_line_list(self, line_table, replace=False, return_table=False, show=Tru
             raise ValueError("Line table must have a 'rest' column'")
 
         # Use the redshift of the displayed spectrum if no redshifts are specified
-        if "redshift" not in line_table.colnames:
-            line_table["redshift"] = self.data()[0].spectral_axis.redshift
-        # Make sure the redshift column is a quantity
-        line_table["redshift"] = u.Quantity(line_table["redshift"])
+        if "redshift" in line_table.colnames:
+            warnings.warn("per line/list redshifts not supported, use viz.set_redshift")
```

Comment:
is a warning reasonable here?  This still allows redshift to be passed, its just ignored.  We also allow _any_ column name to be passed and stored internally (and then retrieved via `specviz.spectral_lines`).  We could instead raise errors for any non-used columns to avoid the impression that they're being used internally?